workers/send_mail/main.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import smtplib
+import traceback
+
+from conductor.ConductorWorker import ConductorWorker
+
+import config
+
+logging.basicConfig()
+log = logging.getLogger('conductor-workers.dummy')
+log.setLevel(logging.INFO)
+
+conf = config.get_config()
+
+
+def send_email(recipient, title, body):
+    message = 'Subject: {}\n\n{}'.format(title, body)
+    server = smtplib.SMTP(config.smtpaddress)
+    server.ehlo()
+    server.starttls()
+    server.login(config.smtplogin, config.smtppassword)
+    server.sendmail(config.emailsender, recipient, message)
+    server.quit()
+
+
+def send_email_task(task):
+    try:
+
+        send_email(task['inputData']['email'], task['inputData']['title'], task['inputData']['body'])
+
+        # always return this well formed response- status, output, logs
+        return {'status': 'COMPLETED',
+                'output': {},
+                'logs': []}
+    except Exception as e:
+        log.fatal(traceback.format_exc(e))
```

Comment:
it's `format_exc()` (without param)

this will throw and thread will silently die forever.

Reply:
also `except: ...` is good enough here